documentation/release_notes.rst

```diff
@@ -8,6 +8,49 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.2.0
+===============
+
+New Features
+------------
+- Added ``sort_by_key`` algorithm. 
```

Comment:
```suggestion
- Added ``sort_by_key`` algorithm for key-value sorting.
```

Note also that this new algorithm needs to be documented, as well as proposed for adding to the specification.

Reply:
For the doc - agree.
But for the spec  - there is no any mention about algo names.